### PR TITLE
Release v0.10.2 (take 4)

### PR DESCRIPTION
**Pull request by kevinbarabash**
Even though this release includes #1951 which disables \includegraphics, we removed the feature completely in the 0.10.1 release so it's not actually a breaking change. This release includes a fix for \not, many MathML improvements, some new symbols, and some changes to the website.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+- Improve MathML for \colorbox and \fcolorbox (#1914)
+- Improve MathML for environments (#1910)
+- Improve MathML for \genfrac barline (#1925)
+- Support \textup and \textmd (#1921) 
+- Improve MathML for \not (#1923)
+- Improve MathML for \Bbbk (#1930)
+- Prevent inadvertent tall delims (#1948)
+
+ ### Removed
+- Disable \includegraphics until trust settings is merged (#1951)
```
Comment (edemaine) [inline CHANGELOG.md]: Perhaps worth mentioning the PR that added `\includegraphics` too? This release simultaneously does both, but only the removal is mentioned in the changelog (unlike the PR description which mentions both).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+- Upgrade minimum development Node version to v8 (#1861)
+- Disable @babel/env debug (#1874)
+- Add issue templates (#1862)
+- Added 'katex-element' (#1905)
+- Fix Users' logo and url (#1896)
+- Load fonts before running screenshotter (#1891)
+- Add Browserstack logo (#1879)
+- Added Android library (#1943)
+- Move custom colors used by Khan into macros.js (#1933)
+- Remove redundant symbols from #1947 (#1954)
```
Comment (edemaine) [inline CHANGELOG.md]: Not important, but we could just list this under the `Add symbol double square brackets` bullet like `(#1947, #1954)`.

**Comment by ice1000** [comment]
When will you people release 0.10.2? Can't wait to see it.

File: CHANGELOG.md (hunk 1 above)
Comment (kevinbarabash) [reply CHANGELOG.md]: I'll modify this to say: `Re-added code for \includegraphics and disabled it until...`

File: CHANGELOG.md (hunk 2 above)
Comment (kevinbarabash) [reply CHANGELOG.md]: Will do.

**Comment by kevinbarabash** [comment]
Sorry about the delay.  The 0.10.2 release has been deployed.  See https://github.com/KaTeX/KaTeX/releases/tag/v0.10.2 for details.

**Pull request merged**